py/client/pydeephaven/_table_ops.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, table):
 
 
 class TimeTableOp(TableOp):
-    def __init__(self, start_time: int = 0, period: int = 1000000000):
+    def __init__(self, start_time: Union[int, str] = 0, period: Union[int, str] = 1000000000):
```

Comment:
I see a few problems here:
* In the python API, we put period first in the args list, because it should never have a default.  Here it is second AND has a default.
* The default zero start time argument is problematic, because 0 nanoseconds is a valid but unusual start time.  The default argument should be some form of "now"

Suggestions:
1) reverse the argument order.
2) remove defaults for period
3) pick a different null value for start time.

@jmao-denver should comment.

Reply:
I discussed this with @jmao-denver and this is the internal API.  We supply `None` as a default argument in the user-facing API in `session.py`. This default is never used.

Reply:
In particular, the default period bothers me.  The default value there is completely arbitrary and users will always specify a value they want.

Reply:
I have removed the internal defaults and now handle the `None` value by setting `start_time = 0`. 

The order of the parameters matches the GRPC member structure, it can be changed but unsure the benefit it will bring.